python/ray/serve/utils.py

```diff
@@ -107,6 +107,74 @@ def parse_request_item(request_item):
     return request_item.args, request_item.kwargs
 
 
+class LoggingContext:
+    r"""
+    Context manager to manage logging behaviors within a particular block, such as:
+    1) Overriding logging level
+    2) Pass in a logging handler 
+        - Configure if handler should be closed afterwards 
+
+    Source (python3 official documentation)
+    https://docs.python.org/3/howto/logging-cookbook.html#using-a-context-manager-for-selective-logging
+    """
+
+    def __init__(self, logger, level=None, handler=None, close=True):
```

Comment:
let's just drop handler and close? We are not using them and we can always add them back in the future.